tests/test_gpu_openmp.py

```diff
@@ -22,17 +22,17 @@ def test_basic(self):
         assert len(trees) == 1
 
         assert trees[0][1].pragmas[0].value ==\
-            'omp target teams distribute parallel for collapse(3)'
-        assert op.body[1].header[0].value ==\
+            'omp target teams distribute parallel for device(devicenum) collapse(3)'
+        assert op.body[2].header[0].value ==\
```

Comment:
why is this and the below ones changing?

Reply:
oh, is it because of the new initialization bit?

if so, then we need a new test, at the very beginning, testing that initialization is producing the right code

Reply:
```
(Pdb) print(op.body[1])
float (*restrict u)[u_vec->size[1]][u_vec->size[2]][u_vec->size[3]] __attribute__ ((aligned (64))) = (float (*)[u_vec->size[1]][u_vec->size[2]][u_vec->size[3]]) u_vec->data;
```

Reply:
yeah, and `op.body[0]` is most likely the initialization code

Reply:
That's right:
```
(Pdb) print(op.body[0])
/* Begin of OpenMP5.0 setup */
int devicenum = 0;
/* End of OpenMP5.0 setup */
```

Reply:
reminder:

> if so, then we need a new test, at the very beginning, testing that initialization is producing the right code

Reply:
Forgot to say it is already implemented locally, and I'll push that together with the next changes.

Reply:
is this now pushed? can I resolve this comment?

Reply:
Yes. It is pushed.